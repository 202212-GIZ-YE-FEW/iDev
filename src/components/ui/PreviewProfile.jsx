function PreviewProfile() {
    return (
        <div className='container relative :w-full h-full  my-5 flex justify-center'>
            <div
                className='absolute bg-cover w-80 h-80 bg-center'
                style={{ backgroundImage: "url(/profile-icon.png)" }}
            >
                <div className='absolute w-20 h-20 bottom-1 top-64 left-32 bg-white border-2 border-black rounded-full'>
                    <div
                        className='absolute left-3 top-2 bg-center bg-cover w-14 h-14'
                        style={{
                            backgroundImage: "url(/edit_profile_icon.png)",
                        }}
                    ></div>
                </div>
            </div>
        </div>
    );
}

export default PreviewProfile;
