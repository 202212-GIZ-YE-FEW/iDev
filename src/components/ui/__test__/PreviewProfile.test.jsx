import renderer from "react-test-renderer";

import PreviewProfile from "../PreviewProfile";

it("renders correctly", () => {
    const tree = renderer.create(<PreviewProfile />).toJSON();
    expect(tree).toMatchSnapshot();
});
