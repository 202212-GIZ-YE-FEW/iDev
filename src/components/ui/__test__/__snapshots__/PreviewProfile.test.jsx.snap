// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    style={
      Object {
        "backgroundImage": "url(/profile-icon.png)",
        "backgroundSize": "cover",
        "bottom": "58.31%",
        "height": "315px",
        "left": "9.51%",
        "position": "absolute",
        "right": "68.61%",
        "top": "16.77%",
        "width": "315px",
      }
    }
  >
    <div
      style={
        Object {
          "background": "#FFFFFF",
          "border": "2px solid #000000",
          "borderRadius": "50%",
          "bottom": "56.72%",
          "height": "70px",
          "left": "17.43%",
          "position": "absolute",
          "right": "35.36%",
          "top": "90.74%",
          "width": "75px",
        }
      }
    >
      <div
        style={
          Object {
            "backgroundImage": "url(/edit_profile_icon.png)",
            "backgroundSize": "cover",
            "height": "52px",
            "position": "absolute",
            "right": "6.12%",
            "top": "13.21%",
            "width": "54px",
          }
        }
      />
    </div>
  </div>
</div>
`;
